src/component/Text.js

```diff
@@ -79,28 +85,31 @@ class Text extends Component {
           this.wordsWithComputedWidth = wordsWithComputedWidth;
           this.spaceWidth = spaceWidth;
         } else {
-          this.updateWordsWithoutCalculate(props);
-
-          return;
+          return this.getWordsWithoutCalculate(props);
         }
       }
 
-      const wordsByLines = this.calculateWordsByLines(
+      return this.calculateWordsByLines(
         this.wordsWithComputedWidth,
         this.spaceWidth,
         props.width
       );
-      this.setState({ wordsByLines });
-    } else {
-      this.updateWordsWithoutCalculate(props);
     }
+    return this.getWordsWithoutCalculate(props);
   }
 
   updateWordsWithoutCalculate(props) {
     const words = !_.isNil(props.children) ? props.children.toString().split(BREAKING_SPACES) : [];
```

Comment:
this line can be replaced by `this.getWordsWithoutCalculate(props)`

Reply:
That'll cause eslint consistent-return error. Because undefined is returned `(props.width || props.scaleToFit) && !isSsr()` is false.